go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/vrecan/death/v3
 go 1.14
 
 require (
-	github.com/gopherjs/gopherjs v0.0.0-20200217142428-fce0ec30dd00 // indirect
-	github.com/smartystreets/assertions v1.1.1 // indirect
-	github.com/smartystreets/goconvey v1.6.4
+	github.com/gopherjs/gopherjs v0.0.0-20211219123610-ec9572f70e60 // indirect
+	github.com/smartystreets/assertions v1.2.1 // indirect
+	github.com/smartystreets/goconvey v1.7.2
```

Comment:
goconvey brings in a lot of dependencies. Might be nice to eliminate it at some point.